import-core/import-rest/pom.xml

```diff
@@ -5,7 +5,7 @@
 	<parent>
 		<groupId>org.devgateway.importtool</groupId>
 		<artifactId>importtool</artifactId>
-		<version>0.0.18-DEV-SNAPSHOT</version>
```

Comment:
DEV should remain in this PR. Actually it should be 19-snapshot. When we released 18 we forgot to upgrade 19

Reply:
same goes for all poms

Reply:
Hey Julian, I noticed the version issue last week and created this pr: https://github.com/devgateway/iatiimport/pull/34.